server/constants.py

```diff
@@ -69,18 +69,22 @@ class SettingDefault:
                 "url": "",
                 "type": "apikey",
                 "docs_href": (
-                    "https://dataverse.harvard.edu/dataverseuser.xhtml?selectTab=apiTokenTab"
+                    "https://{siteUrl}/dataverseuser.xhtml?selectTab=apiTokenTab"
                 ),
                 "targets": [],
             },
             {
-                "name": "dataonestage2",
+                "name": "dataone",
                 "logo": "",
-                "fullName": "DataONE Sandbox",
+                "fullName": "DataONE",
                 "tags": ["publish"],
                 "url": "",
-                "type": "dataone",
-                "state": "unauthorized",
+                "type": "apikey",
+                "docs_href": (
+                    "https://{siteUrl}/portal/token?action=start&"
+                    "target=https%3A//{siteUrl}/portal/token"
```

Comment:
Why do we need `?action..."? It seems to work fine without it (assuming I've logged into DataONE).

Reply:
I realize we can config this, but a second thought is to point this to the docs instead of portal/endpoint, which will do nothing if the user isn't logged in.

Reply:
`action=start` triggers their OAuth flow if you're not logged in